x/auth/ante.go

```diff
@@ -88,17 +89,21 @@ func processSig(ctx sdk.Context, am sdk.AccountMapper, addr sdk.Address, sig sdk
 	// Check and possibly set pubkey.
 	pubKey := acc.GetPubKey()
 	if pubKey.Empty() {
+		if sig.PubKey.Empty() {
+			return nil, sdk.ErrInternal("public Key not found").Result()
+		}
+		if !reflect.DeepEqual(sig.PubKey.Address(), addr) {
```

Comment:
shouldnt need to use `reflect`. I'll follow up on this

Reply:
Thanks for your review :)